app/lib/adapter.rb

```diff
@@ -0,0 +1,101 @@
+class Adapter
+
+  def self.journal_of_nuclear_medicine_adapter(journal, entry)
+    #rss feed does not list keywords. use url to locate keywords.
+    agent = Mechanize.new
+    agent.get(entry.url)
+    abstract = journal.abstracts.build({
+      journal: journal,
+      title: entry.title,
+      authors: entry.author,
+      url: entry.url,
+      body: Adapter.format_abstract_body(entry.summary)
+    })
+    Adapter.create_keywords(abstract, ".kwd-search", agent)
```

Comment:
You already DRYed up the code I was thinking of in my comment to the last PR, awesome.